packages/core/lib/mnemonics/mnemonic.js

```diff
@@ -11,7 +11,7 @@ const Config = require("@truffle/config");
 const defaultUserConfig = Config.getUserConfig();
 const bip39 = require("ethereum-cryptography/bip39");
 const { wordlist } = require("ethereum-cryptography/bip39/wordlists/english");
-const hdkey = require("ethereumjs-wallet/hdkey");
+const hdkey = require("ethereumjs-wallet/dist/hdkey");
```

Comment:
I don't think you should try to go into the `dist` folder this way. I think you should do
```
const { hdkey } = require("ethereumjs-wallet");
```
Check out their example on the repo https://github.com/ethereumjs/ethereumjs-wallet#hd-wallet-api

Reply:
You're right - all fixed now!